### PR TITLE
added a little helper to convert tag_params to array

### PR DESCRIPTION
**Pull request by andela-oakinniranye**

### CODE REVIEW

File: app/models/tag.rb [hunk 1]
```diff
@@ -45,6 +44,16 @@ def resolution_for(tag_name)
         }
```
Comment (andela-oojewale) [inline app/models/tag.rb]: I can't seem to trace the opening counterparts of these braces. Please help.

Comment (andela-echigbo) [inline app/models/tag.rb]: Expand the contracted line blocks @andela-oojewale

Comment (andela-oojewale) [inline app/models/tag.rb]: Thanks!

**Comment by andela-ooranagwa** [comment]
Can we also include a little helper test for this :stuck_out_tongue_winking_eye:

**Pull request merged**